index.bs

```diff
@@ -1864,8 +1864,8 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
     1. Let |spec| be a new [=trigger spec=] with the following items:
         : [=trigger spec/event-level report windows=]
         :: |defaultReportWindows|
-    1. For each integer |triggerData| between 0 (inclusive) and
-        |defaultTriggerDataCardinality| (exclusive):
+    1. [=set/iterate|For each=] integer |triggerData| of [=the exclusive range|the range=] 0 to
```

Comment:
What's the benefit of not displaying "the exclusive range" explicitly, but show "exclusive" at the end?

Reply:
That is what the Infra spec does: https://infra.spec.whatwg.org/#example-the-range

Reply:
It also makes it more clear that the end of the range is exclusive, rather than the beginning.

Reply:
Got it, thanks.